superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/renderers/DomainValueRenderer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+
+export default class DomainValueRenderer extends Component<
+  {},
+  { cellValue: any }
+> {
+  constructor(props: any) {
+    super(props);
+
+    this.state = {
+      cellValue: DomainValueRenderer.getValueToDisplay(props),
+    };
+  }
+
+  // update cellValue when the cell's props are updated
+  static getDerivedStateFromProps(nextProps: any) {
+    return {
+      cellValue: DomainValueRenderer.getValueToDisplay(nextProps),
+    };
+  }
+
+  render() {
+    return (
+      <span>
+        <a href="http://10.162.232.22:8000/gwwk.html">{this.state.cellValue}</a>
```

Comment:
Whaaat is this? Isn't GWWK dead?

Reply:
These(applies to below comment as well) were old renderers. I copied them over to the new grid although they might be useless. They exist in prod though so technically someone could use this renderer if they had a Domain type